node/pkg/watchers/evm/watcher.go

```diff
@@ -225,7 +238,7 @@ func (w *Watcher) Run(parentCtx context.Context) error {
 			p2p.DefaultRegistry.AddErrorCount(w.chainID, 1)
 			return fmt.Errorf("dialing eth client failed: %w", err)
 		}
-		w.ethConn, err = connectors.NewBlockPollConnector(ctx, baseConnector, finalizers.NewDefaultFinalizer(), 250*time.Millisecond, true, safeBlocksSupported)
+		w.ethConn, err = connectors.NewBatchPollConnector(ctx, baseConnector, 250*time.Millisecond)
```

Comment:
The preview poll connector looks like it optionally supported safe blocks based on `safeBlocksSupported` but `NewBatchPollConnector` will always try to query for finalized, safe, and latest. Is that an issue?